Makefile.common

```diff
@@ -175,6 +175,13 @@ ifdef GOLANGCI_LINT
 	$(GOLANGCI_LINT) run $(GOLANGCI_LINT_OPTS) $(pkgs)
 endif
 
+.PHONY: common-lint-fix
```

Comment:
I'd like to get rid of the `$(GO) list...` and see what will happen.
I think it's a leftover from https://github.com/prometheus/prometheus/commit/f5d54e5be62072c0097fce439afa89ab3e399e4e
golangci-lint should take care of that now?

Reply:
Agreed it should work without the `go list`.

Reply:
I removed that.